core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/protobuf/ProtobufSerializer.java

```diff
@@ -220,8 +220,8 @@ public static TextAnnotationProto writeTextAnnotation(TextAnnotation ta) {
         writeSentences(ta, taBuilder);
 
         for (String viewName : Sorters.sortSet(ta.getAvailableViews())) {
-            if (viewName.equals(ViewNames.SENTENCE))
-                continue;
+//            if (viewName.equals(ViewNames.SENTENCE))
+//                continue;
```

Comment:
Why are these being left in as comments?

Reply:
good catch. Removed and pushed changes.